schema/spacewalk/upgrade/susemanager-schema-4.4.6-to-susemanager-schema-4.4.7/003-oval-taskomatic.sql

```diff
@@ -0,0 +1,52 @@
+--
+-- Copyright (c) 2024 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+
+-- Insert task and map to the Java class
+INSERT INTO rhnTaskoTask (id, name, class)
+SELECT sequence_nextval('rhn_tasko_task_id_seq'), 'oval-data-sync', 'com.redhat.rhn.taskomatic.task.OVALDataSync'
+WHERE NOT EXISTS (SELECT 1
+                  FROM rhnTaskoTask
+                  WHERE name = 'oval-data-sync');
+
+-- Insert bunch
+INSERT INTO rhnTaskoBunch (id, name, description, org_bunch)
```

Comment:
I think you are missing some sql files where this insert needs to be added. This SQL file will only run in the upgrade, but for new installations it's not called. Look at what you have add to rhnTaskoTask.sql file.

Reply:
Thanks, it should be fixed now.